apimatic_core/types/union_types/__init__.py

```diff
@@ -0,0 +1,7 @@
+__all__ = [
+    "union_type",
```

Comment:
union_type is being imported from core-interfaces and is not defined in union_types, Is this import needed here?